test/metabase/test/generate.clj

```diff
@@ -206,7 +208,8 @@
                                   :spec        ::field
                                   :insert!     {:model Field}
                                   :relations   {:table_id [:table :id]}
-                                  :constraints {:table_id #{:uniq}}}
+                                  ;:constraints {:table_id #{:uniq}}
```

Comment:
should this be kept in?

Reply:
No, I think this constraint was actually wrong. There's no need why two generated fields can't belong to the same table.